Description: The output started to include a deprecation warning, which was not
expected.
diff --git a/tests/test-install.t b/tests/test-install.t
index a5e61ad..073045e 100644
--- a/tests/test-install.t
+++ b/tests/test-install.t
@@ -199,6 +199,8 @@ Note: we use this weird path to run pip and hg to avoid platform differences,
 since it's bin on most platforms but Scripts on Windows.
   $ ./installenv/*/pip install --no-index $TESTDIR/.. >> pip.log
     Failed building wheel for mercurial (?)
+    DEPRECATION: A future pip version will change local packages to be built in-place without first copying to a temporary directory. We recommend you use --use-feature=in-tree-build to test your packages with this new behavior before it becomes the default.
+     pip 21.3 will remove support for this functionality. You can find discussion regarding this at https://github.com/pypa/pip/issues/7555.
   $ ./installenv/*/hg debuginstall || cat pip.log
   checking encoding (ascii)...
   checking Python executable (*) (glob)
